### PR TITLE
feat: add new commands for licensemanager

### PR DESCRIPTION
**Pull request by iljaSL**

### CODE REVIEW

**Comment by iljaSL** [comment]
Added new commands to the CLI for the license manager endpoints.

File: cmd/license.go [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+func licenseSet(options licenseOptions) error {
+	api := licensemanager.New(curl())
+
+	err := api.SetLicense(options.licenseKey)
+	if err != nil {
+		return err
+	}
+
+	return err
```
Comment (cuongssh) [inline cmd/license.go]: readability: return nil here or remove the error check above before returning err

File: cmd/license.go [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+	err := api.SetLicenseStatistics(options.optin)
+
+	return err
+}
+
+//
+//
+func licenseDeactivateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "deactivate",
```
Comment (cuongssh) [inline cmd/license.go]: Verbs (sub-command) should match: activate and deactivate, set and unset.

File: cmd/license.go [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+		Short: "Deactivate license",
+		Long:  `Deactivate PrivX license`,
+		Example: `
+	privx-cli license deactivate [access flags]
```
Comment (cuongssh) [inline cmd/license.go]: verbs should match: activate and deactivate, set and unset

**Review by cuongssh — CHANGES_REQUESTED** [review]

**Review by cuongssh — APPROVED** [review]

**Pull request merged**